js/compiler/compiler.go

```diff
@@ -148,34 +165,42 @@ type babel struct {
 	vm        *goja.Runtime
 	this      goja.Value
 	transform goja.Callable
-	mutex     sync.Mutex // TODO: cache goja.CompileAST() in an init() function?
+	m         sync.Mutex
 }
 
 func newBabel() (*babel, error) {
 	var err error
 
-	once.Do(func() {
-		vm := goja.New()
-		if _, err = vm.RunString(babelSrc); err != nil {
-			return
-		}
-
-		this := vm.Get("Babel")
-		bObj := this.ToObject(vm)
-		globalBabel = &babel{vm: vm, this: this}
-		if err = vm.ExportTo(bObj.Get("transform"), &globalBabel.transform); err != nil {
+	onceBabelCode.Do(func() {
+		globalBabelCode, err = goja.Compile("<eval>", babelSrc, false)
```

Comment:
Why `<eval>` as `name`?

Reply:
just what it's used in other places ... we can call it "babel.js" I guess

Reply:
I went with a really long descriptive name just in case

Reply:
OK, I'm not really sure why it's important though :sweat_smile: